CodingChallenge/MusicSearch/MusicSearchRouter.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MusicSearchRouter.swift
+//  CodingChallenge
+//
+//  Created by Edgar Barocio on 6/22/19.
+//  Copyright (c) 2019 Edgar Barocio. All rights reserved.
+//
+//  This file was generated by the Clean Swift Xcode Templates so
+//  you can apply clean architecture to your iOS and Mac projects,
+//  see http://clean-swift.com
+//
+
+import UIKit
+
+@objc protocol MusicSearchRoutingLogic {
+    //func routeToSomewhere(segue: UIStoryboardSegue?)
```

Comment:
Delete commented code?

Reply:
the problems with the templates, will do